<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Check_table</class>
 <widget class="QWidget" name="Check_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="head_lable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="bkg_icons.qrc">:/invagate/icons/header.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>验证单</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>751</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>749</width>
      <height>99</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="bkg_icons.qrc">
       <normaloff>:/invagate/icons/to_left90.png</normaloff>:/invagate/icons/to_left90.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNext">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="bkg_icons.qrc">
       <normaloff>:/invagate/icons/to_right90.png</normaloff>:/invagate/icons/to_right90.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>189</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>入库单号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editID"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>206</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>验收时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>751</width>
     <height>301</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>50</number>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>16</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>17</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>18</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>20</string>
    </property>
   </row>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>货物编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>货物名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>单位</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>应收数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>实收数量</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="bkg_icons.qrc"/>
 </resources>
 <connections/>
</ui>
